ferveo/src/api.rs

```diff
@@ -557,14 +570,9 @@ mod test_ferveo_api {
             // In the meantime, the client creates a ciphertext and decryption request
             let msg = "my-msg".as_bytes().to_vec();
             let aad: &[u8] = "my-aad".as_bytes();
-            let rng = &mut thread_rng();
-            let ciphertext = tpke::api::encrypt(
-                SecretBox::new(msg.clone()),
-                aad,
-                &public_key.0,
-                rng,
-            )
-            .unwrap();
+            let _rng = &mut thread_rng();
```

Comment:
Same as above.

Reply:
It's an unused variable, removing it now.